docs/ide/reference/accessibility-tips-and-tricks.md

```diff
@@ -57,7 +57,16 @@ The Visual Studio IDE has toolbars as do many tool windows. The following keyboa
 |Tool window toolbars|Move focus to the toolbars in a tool window. <br> <br> **NOTE:** This works for most tool windows, but only when the focus is in a tool window. Also, you must choose the SHIFT key before the ALT key. In some tool windows, such as Team Explorer, you must hold the SHIFT key for a moment before choosing the ALT key.|**Shift**+**Alt**|
 |Toolbars|Go to the first item in the next toolbar (when a toolbar has focus).|**Ctrl**+**Tab**|
 
-### Other useful keyboard shortcuts
+## Git Keyboard Shortcuts
+
+The Visual Studio status bar shows controls for quick access to various git features. The following keyboard shortcuts help you access them.
```

Comment:
@luhansen-ms If your team does decide to convert the draft to a mergeable PR, please also make the following changes:

- Change "Git Keyboard Shortcuts" to "Git keyboard shortcuts" (we use sentence casing for headings)
- Change instances of "git" to "Git" (so that the mentions are consistent)

Thanks :)